### PR TITLE
Added docs for the async voice feature on SwiftUI

### PR DESCRIPTION
**Pull request by martinmitrevski**
### 🔗 Issue Links

_Provide all Jira tickets and/or Github issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

File: docusaurus/docs/iOS/swiftui/audio-recording.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Voice Recording
+---
+
+Stream Chat's SwiftUI SDK allows you to record and share async voice messages in your channels. The voice recordings have a built-in attachment type (as defined [here](https://getstream.io/chat/docs/sdk/ios/uikit/guides/working-with-attachments/)).
+
+:::note
+Voice Recordings on SwiftUI are available since version 4.46.0.
```
Comment (testableapple) [inline docusaurus/docs/iOS/swiftui/audio-recording.md:8]: What do you think about adding a link to the release on GitHub (I know it's not released yet, but the link will work after releasing)

File: docusaurus/docs/iOS/swiftui/audio-recording.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        gestureLocation: CGPoint
+) -> some View {
+    CustomRecordingView(viewModel: MessageComposerViewModel, location: gestureLocation)
+```
+
+### Locked View
+
+When the user decides to lock the recording, `LockedView` is presented. It provides the user the following actions:
+- discard the recording - you are back to the initial state.
+- stop the recording - you go into the recording preview state.
```
Comment (testableapple) [inline docusaurus/docs/iOS/swiftui/audio-recording.md:47]: Considering the message above, I think `the user` makes more sense, but not a big deal
```suggestion
- discard the recording - the user is back to the initial state.
- stop the recording - the user goes into the recording preview state.
```

File: docusaurus/docs/iOS/swiftui/audio-recording.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+title: Voice Recording
```
Comment (testableapple) [inline docusaurus/docs/iOS/swiftui/audio-recording.md:2]: What's the difference between `audio-recording.md` and `voice-recording.md`? They even have the same title 🤔

**Review by testableapple — APPROVED** [review]
LGTM 👍
Although I didn't quite get the need of having two identical `.md` files 😄

File: docusaurus/docs/iOS/swiftui/audio-recording.md (hunk 3 above)
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/audio-recording.md]: 🤦 fixed now

File: docusaurus/docs/iOS/swiftui/audio-recording.md (hunk 1 above)
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/audio-recording.md]: done

**Pull request merged**